app/src/main/java/org/mozilla/fenix/theme/FirefoxTheme.kt

```diff
@@ -19,12 +25,80 @@ fun FirefoxTheme(
     darkTheme: Boolean = isSystemInDarkTheme(),
     content: @Composable () -> Unit
 ) {
-    MaterialTheme(
-        content = content,
-        colors = if (darkTheme) darkColorPalette else lightColorPalette
+    val colors = if (darkTheme) darkColorPalette else lightColorPalette
+
+    ProvideFirefoxColors(colors) {
```

Comment:
I followed the following for this https://github.com/android/compose-samples/blob/main/Jetsnack/app/src/main/java/com/example/jetsnack/ui/theme/Theme.kt.

Reply:
I don't necessarily think our Firefox theme will fit the MaterialTheme and it will make sense to start a custom color palette and grow that for our components.